cmsis/TOOLCHAIN_GCC/TARGET_CORTEX_A/cache.S

```diff
@@ -1,94 +0,0 @@
-/* Copyright (c) 2009 - 2012 ARM LIMITED
```

Comment:
We don't  need these files anymore? We once removed them and found out later they are still required (here details https://github.com/ARMmbed/mbed-os/issues/4759)

Reply:
I've got a feedback that the Peach board is passing testing. We can always get this file back if proven needed.